src/style.css

```diff
@@ -1 +1,67 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.page {
+  color: rgba(61, 78, 97, 1);
+  box-sizing: border-box;
+}
+
+.search-bar {
+  margin-top: 20px;
+}
+
+.search-bar__search {
+  box-sizing: border-box;
+  outline: none;
+  width: 100%;
+  height: 100%;
+
+  background-repeat: no-repeat;
+  background-image: url(./images/Search.svg);
+
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  transition: box-shadow 0.3s ease;
+
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  font-style: normal;
+}
+
+.search-bar__search--big {
+  background-size: 19px 19px;
+  background-position: 26px center;
+  padding-left: 62px;
+  font-size: 16px;
+  height: 70px;
+}
```

Comment:
Avoid using tag names for styling. You've used classes for styling here, which is good. Keep this up throughout your CSS.